x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/translation_stats.ts

```diff
@@ -13,7 +13,7 @@ import { SIEM_RULE_MIGRATION_TRANSLATION_STATS_PATH } from '../../../../../commo
 import type { SecuritySolutionPluginRouter } from '../../../../types';
 import { authz } from '../../common/api/util/authz';
 import { withLicense } from '../../common/api/util/with_license';
-import { withExistingMigration } from './util/with_existing_migration_id';
+import { withExistingMigration } from '../../common/api/util/with_existing_migration_id';
```

Comment:
Tested locally and realized that this endpoint does not work right now. We use the existing `withExistingMigration` method (line 38) that pre-checks the existence of the migration. But internally that method uses the `ruleMigrationsClient` and runs that checks against siem migration rules not dashboards. Thus we never find dashboard migrations.